server/src/main/java/org/opensearch/gateway/remote/RemoteDownloadStats.java

```diff
@@ -0,0 +1,36 @@
+/*
+ * SPDX-License-Identifier: Apache-2.0
+ *
+ * The OpenSearch Contributors require contributions made to
+ * this file be licensed under the Apache-2.0 license or a
+ * compatible open source license.
+ */
+
+package org.opensearch.gateway.remote;
+
+import org.opensearch.cluster.coordination.PersistedStateStats;
+
+import java.util.concurrent.atomic.AtomicLong;
+
+public class RemoteDownloadStats extends PersistedStateStats {
```

Comment:
Nit: You need to add javadoc for new classes

Reply:
Done